### ISSUE TITLE
Accessibility: Add a skip to content bypass mechanism for keyboard and screen reader users

### ISSUE THREAD

**Issue by robcrowley** [dddwa/dddperth-website#65]
There is no skip navigation link or alternative. Keyboard and screen reader users will have to navigate through the menu every time they reach a new page.

**Recommendation:** Add a skip to content link as the first item in the body. It can be hidden until it receives keyboard focus, at which point it should become visible. E.g.

`<a href="#content" class="skip">Skip to content</a> <div class="container" id="content">…</div>`

```
a.skip { display: block; position: absolute; top: -3em;}
a.skip:focus { position: relative; top: auto; }
```

**Comment by pzi** [comment]
Resolved by #87
